db/migrate/20160728023902_create_pieces.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+      t.string   :type
+      t.integer  :game_id
+      t.string   :color
+      t.integer  :column_coordinate
+      t.integer  :row_coordinate
+
+      t.timestamps
+    end
```

Comment:
Looks good!

Next thing to do: You'll want to add indexes on the associations.

Take a look at this article: https://tomafro.net/2009/08/using-indexes-in-rails-index-your-associations.

It talks about MySQL, but the concept is the same. Let me know if any questions :). You can add them on to this PR.